### PR TITLE
Handling properly datetime fields

### PR DESCRIPTION
**Pull request by carllopez**

### CODE REVIEW

File: materializecssform/templates/materializecssform/field.html [hunk 1]
```diff
@@ -47,13 +47,33 @@
...
+
+        {% if field.help_text %}
+        <p class="help-block">
+            {{ field.help_text|safe }}
+        </p>
+        {% endif %}
+
+    </div>
+    {% elif field|is_datetime_input %}
+    <div class="input-field col {{ classes.label }} {{ classes.value }} {{ classes.single_value }}">
```
Comment (kalwalkden) [inline materializecssform/templates/materializecssform/field.html:71]: Please describe how to use the datetime-local class in the readme

**Review by kalwalkden — CHANGES_REQUESTED** [review]

File: materializecssform/templates/materializecssform/field.html [hunk 2]
```diff
@@ -47,13 +47,33 @@
...
     </div>
-    {% elif field|is_date_input or field|is_datetime_input %}
+    {% elif field|is_date_input %}
     <div class="input-field col {{ classes.label }} {{ classes.value }} {{ classes.single_value }}">
         {% if classes.icon %}
             <i class="material-icons prefix">{{ classes.icon }}</i>
         {% endif %}
 
-        <input type="date" id="{{ field.auto_id }}" name="{{ field.html_name }}" class="datepicker" value="{% if field.value %}{{ field.value }}{% endif %}" {% include 'materializecssform/attrs.html' %} >
+        <input type="date" id="{{ field.auto_id }}" name="{{ field.html_name }}" value="{% if field.value %}{{ field.value }}{% endif %}" {% include 'materializecssform/attrs.html' %} >
```
Comment (kalwalkden) [inline materializecssform/templates/materializecssform/field.html]: Why remove ```class="datepicker"```? The datetime works as is.

**Comment by kalwalkden** [comment]
@carllopez could you include a screenshot of what this looks like?

File: materializecssform/templates/materializecssform/field.html (hunk 1 above)
Comment (carllopez) [reply materializecssform/templates/materializecssform/field.html:71]: Readme has been updated

**Review by kalwalkden — APPROVED** [review]

**Comment by carllopez** [comment]
@kalwalkden here you go:

datetimefield

**Pull request merged**